### PR TITLE
fix(curriculum): increase readability of the introduction

### PR DESCRIPTION
**Pull request by L4rryFisherman**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by jeremylt** [comment]
Personally I don't see this wording as better, just different.

**Comment by L4rryFisherman** [comment]
Maybe just me then, I had to read the sentence 3/4 times to understand what was meant in the original version.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/generate-an-array-of-all-object-keys-with-object.keys.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ dashedName: generate-an-array-of-all-object-keys-with-object-keys
 
 # --description--
 
-We can also generate an array which contains all the keys stored in an object using the `Object.keys()` method and passing in an object as the argument. This will return an array with strings representing each property in the object. Again, there will be no specific order to the entries in the array.
+We might want to retrieve all keys of our object. Using the `Object.keys()` method we can generate an array which contains strings for all the keys that are stored in an object. To do this, we pass the object as the argument of `Object.keys()`. Again, there will be no specific order to the entries in the array.
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/generate-an-array-of-all-object-keys-with-object.keys.md:11]: If we want to change this section, this is more grammatically straightforward:

```suggestion
We can also generate an array which contains all the keys stored in an object with the `Object.keys()` method. This method takes an object as the argument and returns an array of strings representing each property in the object. Again, there will be no specific order to the entries in the array.
```

**Review by Sboonny — APPROVED** [review]

**Comment by nhcarrigan** [comment]
I'm with @jeremylt on this one. Different wording doesn't necessarily mean better - and changing this would require re-translation so I'm not sure it's worth the overhead.

**Comment by jeremylt** [comment]
I'm on the fence with the new rewording. Now its a small change that might effectively be English only. The updated flow after my suggestion is a bit clearer. It might be a happy-to-glad sort of thing though.

**Review by moT01 — APPROVED** [review]
I think this wording is better, if only slightly - and that's really only my opinion, other people may prefer the current verbiage. I am fine to merge this.

Thanks for contributing to freeCodeCamp @L4rryFisherman 🎉 

In the future, if you have ideas for improvements like this, I recommend creating an issue with your suggestion first.

**Pull request merged**